### PR TITLE
(Ab)using our backend

### PR DESCRIPTION
**Pull request by JScheire**
Listing hackru announcements #48

### CODE REVIEW

File: challenge/src/components/announce/AnnouncementComponent.module.css [hunk 1]
```diff
@@ -0,0 +1,9 @@
+.container {
```
Comment (mjrb) [inline challenge/src/components/announce/AnnouncementComponent.module.css]: nit pick: we can be a bit more specific with this class name, for example `announcements container`

File: challenge/src/components/announce/AnnouncementComponent.module.css [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    height: 800px;
```
Comment (mjrb) [inline challenge/src/components/announce/AnnouncementComponent.module.css]: gitving height in pixes can get wonky on other screens, for example on my laptop this component is to tall so their are now 2 scroll bars. perhaps using % instead may help, or doing `@media` screen queries if you want to be extra fancy
Screenshot from 2020-06-09 23-08-11

File: challenge/src/components/announce/AnnouncementComponent.jsx [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+    state = {announcements: [], loading: true}
+
+    async componentDidMount() {
+        const response = await fetch('https://api.hackru.org/prod/dayof-slack')
+        const json = await response.json();
+        this.setState({
+            loading: false,
+            announcements: json.body.map(a => ({
+                ...a,
+                ts: new Date(parseFloat(a.ts) * 1000).toLocaleTimeString()
```
Comment (mjrb) [inline challenge/src/components/announce/AnnouncementComponent.jsx:14]: I like that you also used and converted the timestamps too :+1:

**Review by mjrb — CHANGES_REQUESTED** [review]
pretty good, just some small style tweaks needed

File: challenge/src/components/announce/AnnouncementComponent.module.css (hunk 2 above)
Comment (JScheire) [reply challenge/src/components/announce/AnnouncementComponent.module.css]: vh seems to do the trick!

**Review by mjrb — APPROVED** [review]
LGMT

**Pull request merged**